lib/tasks/civi_crm.rake

```diff
@@ -63,6 +65,79 @@ namespace :civi_crm do
     end
   end
 
+  namespace :import_by_csv do
```

Comment:
can we better namespace the CSV related tasks please?
```suggestion
  namespace :erc_auth do
    namespace :csv_import do
      namespace :comarcals do
      ....
```

Also, can we extract this into a specific csv.rake file?